src/RustyObjectStore.jl

```diff
@@ -1,6 +1,7 @@
 module RustyObjectStore
 
-export init_object_store, get_object!, put_object, StaticConfig, ClientOptions, Config, AzureConfig, AWSConfig
+export init_object_store, get_object!, put_object, delete_object
+export StaticConfig, ClientOptions, Config, AzureConfig, AWSConfig
```

Comment:
I just realized I forgot to export `put_object_stream`, `WriteStream` and maybe more stuff, could you fix this as part of your PR?